Frends.SFTP.UploadFiles/Frends.SFTP.UploadFiles/Definitions/SingleFileTransfer.cs

```diff
@@ -0,0 +1,537 @@
+﻿using System.Text;
+using Renci.SshNet;
+
+namespace Frends.SFTP.UploadFiles.Definitions
+{
+    /// <summary>
+    /// Class for executing single file transfer.
+    /// </summary>
+    public class SingleFileTransfer
```

Comment:
SingleFileTransfer class cannot be internal because it's been used in SFTPLogger class which also cannot be internal and therefore creates conflict if SingleFileTransfer class is set to internal. I have changed the TransferSingleFile method as internal.

Reply:
I am pretty sure SFTPLogger can also be internal.

Reply:
Fixed